src/toolcalls/chain/chainsRegistry.ts

```diff
@@ -294,14 +432,16 @@ export const chainRegistry: ChainRegistry = {
   },
 };
 
-export const chainNameToolCallParam = {
-  name: 'chainName',
-  type: 'string',
-  description:
-    'name of the chain the user is interacting with, if not specified by the user assume Kava EVM',
-  enum: [
-    ...Object.keys(chainRegistry[ChainType.EVM]),
-    ...Object.keys(chainRegistry[ChainType.COSMOS]),
-  ],
-  required: true,
+export const getChainConfigByName = (
+  chainName: string,
+  chainType: ChainType,
+): unknown => {
```

Comment:
possibly return `ChainConfig` here instead of unknown? then can avoid the type casts, and potential unhandled null option

Reply:
great catch! thanks